classes/ReassuranceActivity.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * 2007-2019 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author PrestaShop SA <contact@prestashop.com>
+ * @copyright  2007-2019 PrestaShop SA
+ * @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ *  International Registered Trademark & Property of PrestaShop SA
+ */
+
+class ReassuranceActivity extends ObjectModel
+{
+    const TYPE_LINK_NONE = 0;
+    const TYPE_LINK_CMS_PAGE = 1;
+    const TYPE_LINK_URL = 2;
+
+    public $id;
+    public $icone;
+    public $icone_perso;
+    public $title;
+    public $description;
+    public $status;
+    public $position;
+    public $id_shop;
+    public $type_link;
+    public $link;
+    public $id_cms;
+    public $date_add;
+    public $date_upd;
+
+
+    /**
+     * @see ObjectModel::$definition
+     */
+    public static $definition = array(
+        'table' => 'psreassurance',
+        'primary' => 'id_psreassurance',
+        'multilang' => true,
+        'multilang_shop' => true,
+        'fields' => array(
+            'icone' => array('type' => self::TYPE_STRING, 'shop' => true, 'validate' => 'isCleanHtml', 'size' => 255),
```

Comment:
```suggestion
            'icon' => array('type' => self::TYPE_STRING, 'shop' => true, 'validate' => 'isCleanHtml', 'size' => 255),
```
Use english word instead, same for icon_perso, use `custom_icon`